src/scenes/Search/store.js

```diff
@@ -68,13 +67,14 @@ export default class SearchStore {
   }
 
   async fetch(phrase) {
-    let result = [];
-    if (_.isEmpty(phrase)) return result;
+    if (_.isEmpty(phrase)) return [];
     let oracles = await searchOracles(phrase);
     oracles = _.uniqBy(oracles, 'txid').map((oracle) => new Oracle(oracle, this.app));
-    let topics = await searchTopics(phrase);
+    const topicFilters = [];
+    topicFilters.push({ status: OracleStatus.WITHDRAW });
+    let topics = await searchTopics(phrase, topicFilters);
```

Comment:
add the `orderBy` here instead of ordering it after you get the return. you will need to change the `seachTopics` query.
```
export function searchTopics(phrase, filters, orderBy, limit, skip) {
}

const orderBy = { field: 'endTime', direction: SortBy.DESCENDING };
let topics = await searchTopics(phrase, topicFilters, orderBy);
```

Reply:
I found that the topic object does not have `endTime`, so I think we don't need to sort withdraw events.